test/input/counter.gr

```diff
@@ -6,6 +6,7 @@ let counter = {
   counter_help
 };
 {
+  # TODO: This should have some equivalent of OCaml's ignore(...)
```

Comment:
I'd get rid of this comment and instead we should just have a discussion about the type of `print` as well as something like `ignore`, and make issues accordingly.